content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -83,43 +100,79 @@ Next, configure the :guilabel:`Fiscal Information` in the :guilabel:`Sales & Pur
 - :guilabel:`Commercial Name`: If the company uses a specific commercial name, and it needs to be
   displayed in the invoice.
 
-Carjaval credentials configuration
-----------------------------------
+Electronic invoice credentials configuration
+--------------------------------------------
 
-Once the modules are installed, the user credentials **must** be configured, in order to connect
-with Carvajal Web Service. To do so, navigate to :menuselection:`Accounting --> Configuration -->
-Settings` and scroll to the :guilabel:`Colombian Electronic Invoicing` section. Then, fill in the
-required configuration information provided by Carvajal:
+Once the modules are installed, the user credentials **must** be configured in order to connect
+with |DIAN|'s web service. To do so, navigate to :menuselection:`Accounting --> Configuration -->
+Settings` and scroll to the :guilabel:`Colombian Electronic Invoicing` section. Then, complete the
+following configuration:
 
-- :guilabel:`Username` and :guilabel:`Password`: Username and password (provided by Carvajal) to the
-  company.
-- :guilabel:`Company Registry`: Company's NIT number *without* the verification code.
-- :guilabel:`Account ID`: Company's NIT number followed by `_01`.
-- :guilabel:`Colombia Template Code`: Select one of the two available templates (`CGEN03` or
-  `CGNE04`) to be used in the PDF format of the electronic invoice.
+- Select :guilabel:`DIAN: Free Service` as the :guilabel:`Electronic Invoicing Provider`.
+- Configure the :guilabel:`Operation Modes` for the respective types of documents (*electronic
+  invoices* or *support documents*) to be generated from Odoo:
 
-Enable the :guilabel:`Test mode` checkbox to connect with the Carvajal testing environment.
+  - :guilabel:`Software Mode`: the type of document to be generated with the operation mode.
+  - :guilabel:`Software ID`: the ID generated by |DIAN| for the specific operation mode.
+  - :guilabel:`Software PIN`: the PIN selected in the operation mode configuration in the |DIAN|
+    portal.
+  - :guilabel:`Testing ID`: the testing ID generated by |DIAN| and obtained after testing the
+    operation mode.
 
-Once Odoo and Carvajal are fully configured and ready for production, deactivate the :guilabel:`Test
-mode` checkbox to use the production database.
+- Configure the available :guilabel:`Certificates` to sign your electronic documents:
 
-.. image:: colombia/carvajal-configuration.png
-   :align: center
-   :alt: Configure credentials for Carvajal web service in Odoo.
+  - :guilabel:`Name`: the name of the certificate.
+  - :guilabel:`Certificate`: upload the certificate file in PEM format. After uploading a valid
+    certificate file, the following fields display:
 
-.. important::
-   :guilabel:`Test mode` must **only** be used on duplicated databases, **not** the production
-   environment.
+    - :guilabel:`Certificate Password`: the password to decrypt the PKS file.
```

Comment:
```suggestion

```
@samueljlieber this passwork is not needed for Colombia, since the certificates for this country can only be decoded by a PEM key file